test/built-ins/String/prototype/isWellFormed/returns-boolean.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2022 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-String.prototype.at
+description: >
+  The method should return an Iterator instance.
+info: |
+  String.prototype.isWellFormed ( )
+
+  1. Let O be ? RequireObjectCoercible(this value).
+  2. Let S be ? ToString(O).
+  3. Return IsStringWellFormedUnicode(S).
+
+features: [String.prototype.isWellFormed]
+---*/
+assert.sameValue(typeof String.prototype.isWellFormed, 'function');
+
+var leadingPoo = '\uD83D';
+var trailingPoo = '\uDCA9';
+var wholePoo = leadingPoo + trailingPoo;
+
+assert.sameValue(
+  ('a' + leadingPoo + 'c' + leadingPoo + 'e').isWellFormed(),
+  false,
+  'leading lone surrogates are not well-formed'
+);
+assert.sameValue(
+  ('a' + trailingPoo + 'c' + trailingPoo + 'e').isWellFormed(),
+  false,
+  'trailing lone surrogates are not well-formed'
+);
+assert.sameValue(
+  ('a' + leadingPoo + trailingPoo + 'd').isWellFormed(),
```

Comment:
This is backwards. Well, not backwards. You know what I mean.

Reply:
whoops, on it